setup.cfg

```diff
@@ -1,2 +1,13 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [bdist_wheel]
 universal = 1
+
+[flake8]
+doctests = 1
```

Comment:
For which plugin is this used and what does this mean? I never saw that before.

Reply:
This advises `flake8` to check the doctests, too. Previously (aka some month ago) we used the command line parameter `--doctests`.

Reply:
It's the pyflakes plugin of flake8, if that's what you meant.  It means that names imported at the module scope and used only inside doctests don't trigger false positives, among other things.

Reply:
Thanks!